doc/src/documentation/architecture/adr/adr006-kiss-frontend-components.md

```diff
@@ -0,0 +1,25 @@
+# ADR-006 frontend components without complex logic
+
+## Status
+
+<adr-status status='accepted' />
+
+## Context
+
+By components are meant here vue components, in the form of single-file components. These contain a template,
+a script part and styling.
+
+## Decision
+
+There should be no functional logic in components. The code of a component should be limited to databinding.
+The logic, apart from databinding, is settled accomplished by composables or services.
+
+### Todo
+
+- components of open pull requests have to be adopted
+- at the time of the decision there were no completed PRs so no existing code is to be adapted
+
+## Consequences
+
+The logic is easier to test because it can be tested independently of the components. The logic may need to be
```

Comment:
independently *from*

Reply:
IntelliJ says `of` is the usual collocation. Are u sure that `from` is correct?

Reply:
IntelliJ says `of` is the usual collocation. Are u sure that `from` is correct?

Reply:
I just saw that both is fine. You can leave the *of*